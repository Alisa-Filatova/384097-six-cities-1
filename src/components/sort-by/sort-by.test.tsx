import * as React from 'react';
import * as renderer from 'react-test-renderer';
import SortBy from './sort-by';
import SortType from '../../types/enums/sort-type';

describe(`SortBy`, () => {
  it(`renders correctly`, () => {
    const sortBy = renderer.create(
        <SortBy
          currentItem={SortType.POPULAR}
          onToggle={jest.fn()}
          onLowToHighClick={jest.fn()}
          onHighToLowClick={jest.fn()}
          onPopularClick={jest.fn()}
          onTopRatedClick={jest.fn()}
          isOpen={true}
        />
    ).toJSON();
    expect(sortBy).toMatchSnapshot();
  });
});
