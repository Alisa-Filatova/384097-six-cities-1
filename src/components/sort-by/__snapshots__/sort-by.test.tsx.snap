// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SortBy renders correctly 1`] = `
<form
  action="#"
  className="places__sorting"
  method="get"
  onClick={[Function]}
>
  <span
    className="places__sorting-type"
    tabIndex={0}
  >
    <span
      className="places__sorting-caption"
    >
      Sort by
    </span>
     
    Popular
    <svg
      className="places__sorting-arrow"
      height="4"
      width="7"
    >
      <use
        xlinkHref="#icon-arrow-select"
      />
    </svg>
  </span>
</form>
`;
